README.md

```diff
@@ -35,9 +33,15 @@ Prerequisites:
       in [`mastodon/mastodon.env.template`](mastodon/mastodon.env.template).
       - Make sure the SMTP configuration is correct as this is the only way of using the initial admin account without manually logging
         into the instance to retrieve the password.
-   4. Lastly, set up a login (username and password) for your monitoring dashboards at [`monitoring/monitoring-admin.env`](monitoring/monitoring-admin.env).
+   4. Set up a login (username and password) for your monitoring dashboards at [`monitoring/monitoring-admin.env`](monitoring/monitoring-admin.env).
       You can add more users and change your password later via the Grafana web user interface.
-3. Generate Ignition deployment configuration based on your setting by running
+   5. Generate a ssh keypair for remote backups. The keys are also required for node cloning.
+      ```shell
+      ssh-keygen -P "" -f backup/backup_key
```

Comment:
An empty password? I can see in `man ssh-keygen` a note that host keys must have an empty passphrase. Is it the case here? Could you maybe write a sentence why it's acceptable?

Reply:
Hmmm, that's a good point. I've done it this way so I can use the key in clone-node.sh, which runs at the target node during provisioning.

If there's a way to feed the key's passphrase into SSH in [`clone-node.sh`](https://github.com/flatcar/flatcar-mastodon/pull/6/files#diff-a7349505e7b8491b0ae5f7aa1c8b69920d56d8cee11d6669da7214e970cbe974R18) (which runs during provisioning of the target node) then there's no reason not to use one.